karma.conf.js

```diff
@@ -0,0 +1,39 @@
+module.exports = function (config) {
+  const jqueryFile = process.env.USE_OLD_JQUERY === 'true' ? 'js/tests/vendor/jquery-1.9.1.min.js' : 'assets/js/vendor/jquery-slim.min.js'
```

Comment:
Maybe `process.env.USE_OLD_JQUERY` would be enough?

Reply:
I don't understand what you mean here 😟

Reply:
I was wondering, wouldn't this be enough?

```js
const jqueryFile = process.env.USE_OLD_JQUERY ? 'js/tests/vendor/jquery-1.9.1.min.js' : 'assets/js/vendor/jquery-slim.min.js'
```

Oh, and perhaps we should only change the filename and keep the path since it's common?